paf-mvp-core-js/src/monitoring/tracer-factory.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  trace,
+  Tracer,
+  Span,
+  SpanOptions,
+  SpanContext,
+  SpanStatus,
+  Attributes as SpanAttributes,
+  AttributeValue as SpanAttributeValue,
+  TimeInput,
+  Context,
+  Exception,
+} from '@opentelemetry/api';
+
+class NoopSpanContext implements SpanContext {
```

Comment:
This doesn't seem to be used anywhere (I don't find usage of `spanContext()` method in `NoopSpan` class). Can you give some context?

Reply:
I've implemented just exactly the interface specified by OpenTelemetry.
To be discussed